vmm/src/lib.rs

```diff
@@ -3388,6 +3460,41 @@ mod tests {
             .is_some());
     }
 
+    #[test]
+    fn test_configure_vmm_from_json() {
+        let mut vmm = create_vmm_object(InstanceState::Uninitialized);
+
+        let kernel_file = NamedTempFile::new().unwrap();
+        let rootfs_file = NamedTempFile::new().unwrap();
+
+        let json = String::from(
+            r#"{
+                    "boot_source":{
+                        "kernel_image_path": ""#,
```

Comment:
Why not `format!` the string? The alignment is weird and it's not very readable.

Reply:
I agree, it looks better.